src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,22 @@
+package calculator;
+
+import calculator.enums.Sign;
+import calculator.util.StringUtil;
+
+public class StringCalculator {
+  private static final int FIRST_KEY = 0;
+  private static final int STEP = 2;
+  private static final int START_INDEX = 1;
+  private static final int NEXT_INDEX = 1;
+  public int calculate(String text) {
```

Comment:
가독성을 위해 공백 라인을 추가해보면 좋을 것 같습니다.
```suggestion
  private static final int NEXT_INDEX = 1;
  
  public int calculate(String text) {
```